code/game/objects/items/devices/scanners.dm

```diff
@@ -234,10 +300,10 @@ SLIME SCANNER
 		else if(!heart)
 			msgs += "<span class='notice'><font color='red'><b>Subject has no heart.</b></font>"
 
-	if(H.getStaminaLoss())
+	if(H.getStaminaLoss() || HAS_TRAIT(user, TRAIT_MED_MACHINE_HALLUCINATING) && prob(5))
 		msgs += "<span class='notice'>Subject appears to be suffering from fatigue.</span>"
 
-	if(H.getCloneLoss())
+	if(H.getCloneLoss() || HAS_TRAIT(user, TRAIT_MED_MACHINE_HALLUCINATING) && prob(5))
```

Comment:
wrap these condiationals please
```suggestion
	if(H.getCloneLoss() || (HAS_TRAIT(user, TRAIT_MED_MACHINE_HALLUCINATING) && prob(5)))
```

Reply:
I don't think these parens are necessary semantically, but I can do it if you want for clarity

Reply:
I'm not sure, byond might read it as `(H.getCloneLoss() || HAS_TRAIT(user, TRAIT_MED_MACHINE_HALLUCINATING)) && prob(5)` otherwise. Either way, its better to make it easier to read. I generally advise doing it whenever mixing || and &&